### ISSUE TITLE
Question on hide behaviour

### ISSUE THREAD

**Issue by NicoFerna** [enyo/opentip#21]
Hi,
Is it currently possible to hide the tip on mouseout unless the cursor goes over the tooltip? In other words, keep the tooltip open if the user mouseovers from the trigger to the tooltip.

EDIT: I have tried with setting hideTrigger:'tip' and hideOn:'mouseout' but the tip still disappears when I mouseout of the trigger it seems... 

Cheers,
Nicolas

**Comment by enyo** [comment]
Hi! That's actually this issue: #6
I'll move it to the top of my todo list and hope I find the time to fix it tomorrow.
Cheers

**Comment by NicoFerna** [comment]
OK, looking at issue #6 I understand why this wasn't working as intended. Thanks for looking into it, I would really appreciate to have this option to work.

**Comment by enyo** [comment]
Issue #6 is now fixed, so I'll close this. I'll make a release in a few minutes.

**Comment by enyo** [comment]
Ok. I released it under 2.2.2:

https://github.com/enyo/opentip/tree/v2.2.2

**Comment by zeleniy** [comment]
So how i should define opentip in this case - not hide tip if mouse cursor over/under it or over/under target, and hide it in all other cases - especially if i do it programmatically:

```
this.contextMenu = new Opentip(target, {
  showOn: null,
  delay: 0.0,
  hideDelay: 0.0,
  hideTriggers: ["closeButton"],
  group: "tooltip-fake-group"
});
```

?
